opendebates/utils.py

```diff
@@ -131,6 +131,8 @@ def sort_list(citations_only, sort, ideas):
         ideas = ideas.order_by("votes")
     elif sort == "-local_votes":
         ideas = ideas.order_by("-local_votes")
+    elif sort == "-current_votes":
+        ideas = ideas.order_by("-current_votes")[:30]
```

Comment:
I didn't think this was going to be limited to 30 in the code -- is this correct per the client?